### ISSUE TITLE
TLS Support with system tests

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#671]
TLS Support is implemented however there are some issues with the system test setup which required a workaround of setting both peers to be gossip leaders which should not be setup that way. The issue is tracked by https://jira.hyperledger.org/browse/FAB-2942
Also it is possible to enable the system tests to run in TLS mode as follows, it is currently disabled.
To change them to tls update the travis.yaml file as follows
```
       - env: SYSTEST=hlfv1-1_tls SYSTEST_HLF=hlf FC_TASK=systest
       - env: SYSTEST=hlfv1-2_tls SYSTEST_HLF=hlf FC_TASK=systest
```

**Comment by sstone1** [comment]
@davidkel is this something we could turn on now, and turn off the non-TLS runs?

**Comment by davidkel** [comment]
@sstone1 reason I hadn't done it yet was that to make it work I had to make both peers gossip leaders and when I asked if that was the right thing to do via JIRA, they asked for checking of stuff which I did and it all looked ok so didn't get clarification that this was correct. Now they say they can't recreate it. So I will try again with alpha2 and see if the problem is resolved. If it is then I will enable them.
